### PR TITLE
Jun Yin's pull requests

### PR DESCRIPTION
**Pull request by jjeryin**
OpenROAD design contest

### CODE REVIEW

File: flow/logs/asap7/ibex/base/2_2_floorplan_io.log
Comment (maliberty) [inline flow/logs/asap7/ibex/base/2_2_floorplan_io.log:1]: Log files are never committed.  I don't even know how  you got this into git as we have flow/logs in the .gitignore.

Comment (jjeryin) [reply flow/logs/asap7/ibex/base/2_2_floorplan_io.log:1]: I just manually uploaded to my forked repo first and request a pull after the upload was done.

Comment (maliberty) [reply flow/logs/asap7/ibex/base/2_2_floorplan_io.log:1]: Yes but how did you commit it to your repo?  It should have the same .gitignore.  In any case there should be no files in the PR that .gitignore covers.

**Comment by vijayank88** [comment]
@jjeryin 
Follow design contest channel instruction, once submission finalized, let you know the instruction for PR creation.

**Pull request closed without merging**